themes/default/content/blog/customer-managed-deployment-agents-launch/index.md

```diff
@@ -0,0 +1,138 @@
+
+## Deploy on Your Terms: Launching Customer-Managed Agents
+
+With the launch of Customer-Managed Agents, you can now self-host the deployment agents within your infrastructure, bringing the same power and flexibility of Pulumi-hosted deployments to isolated environments. Customers with hard security and compliance requirements can now fully leverage Pulumi Deployments. 
+
+## Key Benefits of Customer-Managed Deployment Agents
+
+- **Host anywhere**: You can host the deployment agents anywhere to manage infrastructure, even within your fully private VPCs
+- **Any hardware, any environment<sup>1</sup>**: Run the agents on any hardware of your choice and configure the environment that meets your needs
+- **Mix & match**: You can use standard Pulumi-hosted deployments for your development stacks and use self-hosted Customer-Managed Agents for your private network infrastructure. You can mix and match to suit your unique needs
+- **Multiple pools**: You can set up multiple deployment agent pools, assign stacks to specific pools, and scale agents dynamically to increase your deployment concurrency. Customers can have up to 150 concurrent deployments
+- **Meet compliance**: You can configure the agents with the credentials needed to manage your infrastructure. This way your cloud provider credentials never leave your private network
+
+<sup>1</sup> *Currently Linux and MacOS are supported*
+
+Customer-Managed Agents support all the deployment triggers currently offered by Pulumi Deployments such as click to deploy, the Pulumi Deployments REST API, git push to deploy, Review Stacks, and remote Automation API. 
+
+{{% notes "info" %}}
+Customer-Managed Agents is available on the Business Critical edition of Pulumi Cloud. [Contact sales](/contact/?form=sales) if you are interested and want to enable Customer-Managed Agents. If you are a self-hosted Pulumi Cloud customer, please [get in touch](https://share.hsforms.com/1YajiJ73sSuGn5RoTkyvKxg2mxud) to be notified when it is available.
+{{% /notes %}}
+
+
+## How to get started
+
+Before you begin, ensure you have installed the [Pulumi Github App](/docs/using-pulumi/continuous-delivery/github-app/) and update the [source control settings](/docs/pulumi-cloud/deployments/get-started). [Docker](https://docs.docker.com/engine/) is a requirement for the agent.
+
+### Install and configure the agent
+
+Go to **Deployment runners** under Organization Settings in the left-hand navigation menu. Once on the Deployment runners page, click on the 'Add a new pool' button. Next, provide a name and description for your agent. You will then get a new access token for the pool, ensure to copy and save the token in a secure location.
+
+{{< video title="Create a new pool" src="./create-new-pool.mp4" autoplay="true" loop="true" >}}
```

Comment:
These videos move _really_ fast -- so fast that it's hard to really see what's going on in them without watching them several times and looking super closely, which is a little disorienting. If there's any way to slow them down, I'd suggest it.